tests/nvme_test.py

```diff
@@ -309,12 +309,12 @@ def test_connect_single_ns(self):
         # nothing to connect to
         with self.assertRaisesRegexp(GLib.GError, r"Error connecting the controller: "):
             # FIXME: libnvme itself spits an error on stdout
-            BlockDev.nvme_connect(self.SUBNQN, 'loop', None, None, None, None, self.HOSTNQN, None)
+            BlockDev.nvme_connect(self.SUBNQN, 'loop', None, None, None, None, self.HOSTNQN, None, None)
```

Comment:
Please add override for this function to `BlockDev.py` so users don't need to specify `None` for extra arguments.

Reply:
Added, good idea.